### ISSUE TITLE
Preview of available templates

### ISSUE THREAD

**Issue by rskikuli** [phpList/phplist-templates#1]
I would propose to provide with images (in PNG format) of the templates. This will make it easier for anyone to have a preview.

**Comment by kristicunga** [comment]
Hello, I worked on this and created previews for each template with two different preview formats: 
https://cloud.collective68.tech/s/nSeLeTXaZWjR8sa

Hopefully this helps phpList users have a better understanding of the html templates available for phpList.

**Comment by kristicunga** [comment]
Hello, I worked on this and created previews for each template with two different preview formats: 
    
- Template image only preview: https://cloud.collective68.tech/s/WG9PZZqbemEo9ck
- Laptop mockup preview: https://cloud.collective68.tech/s/nSeLeTXaZWjR8sa
    
Hopefully this helps phpList users have a better understanding of the html templates available for phpList.

**Comment by samtuke** [comment]
To clarify: the benefit of these images is that they show the admin what the existing phpList templates look like in completion, instead of only showing the separate parts as they are presented in the template and campaign content editor?

I think that's a great benefit, and yes these should be added to the UI somewhere. I suppose we would need a new location for the images, and a database record to associate the preview image with the corresponding template.

**Comment by kristicunga** [comment]
> To clarify: the benefit of these images is that they show the admin what the existing phpList templates look like in completion, instead of only showing the separate parts as they are presented in the template and campaign content editor?

Yes, this is exactly what we had in mind. Hopefully this will help everyone who is using the templates, understand how they look.

> I think that's a great benefit, and yes these should be added to the UI somewhere. I suppose we would need a new location for the images, and a database record to associate the preview image with the corresponding template.

I'm not sure if I have the know-how to help on this, but it will be great if it could be implemented. And of course, we will be more than happy to see this implemented.
